tests/fmt.rs

```diff
@@ -0,0 +1,20 @@
+#[test]
+fn fmt() {
```

Comment:
Does this mean that `./util/dev fmt --check` will be run as part of `cargo test`?

If so, does it output the correct command to run to fix formatting issues upon test failure?

Reply:
> Does this mean that ./util/dev fmt --check will be run as part of cargo test?

Yes

> If so, does it output the correct command to run to fix formatting issues upon test failure?

It tells you to run `./util/dev fmt` which should fix it.

Reply:
Actually, it's part of the stdout that gets written out when the test fails. I guess I could add it in the assert.